routes/api.js

```diff
@@ -10,13 +10,14 @@ var config = require('../app/config');
 var Promise = require('es6-promise').Promise;
 var url = require('url');
 
-var KERNEL_GATEWAY_URL = config.get('KERNEL_GATEWAY_URL');
+var kgUrl = config.get('KERNEL_GATEWAY_URL');
+var kgAuthToken = config.get('KG_AUTH_TOKEN');
```

Comment:
Set a false-y default (blank string or null) in `config.json` which is serving as documentation of all of the defaults and optional settings when you run `make help`.